### ISSUE TITLE
Simultaneous simulation break more after config_simul.yml

### ISSUE THREAD

**Issue by paugier** [exabl/snek5000#116]
Similar to #53.

After #113, a file `config_simul.yml` is created for each simulation. Since the config file is marked as an input for the target `build_third_party`, the third parties are rebuild for every simulation.

This leads to random bugs when simulations run simultaneously. For example, things like:

```
cat build.log 
ar -cr libblasLapack.a *.o
mkdir: cannot create directory ‘gslib’: File exists
```

I pushed a bad workaround in branch [fix/build_third_party](https://github.com/exabl/snek5000/tree/fix/build_third_party) (2773f2fbec8d5044589b059551d145a4f5e1ce4f) but I think it would be a good time to implement a good solution to fix at the same time #53 and this issue.

**Comment by ashwinvis** [comment]
Good catch. I agree this is high time to fix #53. I will see if a pure Python solution is possible.
